src/actions/profileActions.js

```diff
@@ -12,6 +12,13 @@ export const fetchProfileSuccess = (profile) => {
   };
 };
 
+export const addMango = (mangoAmount) => {
+  return {
+    type: "ADD_MANGO",
```

Comment:
The feed reducer also has an "ADD_MANGO" action type that takes in a different payload which could cause a crash.

Reply:
Ah i see, I repurposed the existing "ADD_MANGO" case in the userProfileReducerDB but yeah, best to rename. If we have time, we should go over all our reducer cases and find a more uniform and specific naming system cause it's a bit of a mess right now.